client/app/src/main/java/com/example/mrides/Notification/InboxAdapter.java

```diff
@@ -23,28 +24,25 @@ public InboxAdapter(Context inboxContext){
     }
 
     public void setViewComponents(Map<String,String> body){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.